### DISCUSSION TITLE
Never minimize button

### DISCUSSION THREAD

**Discussion by Panostis** [WindowTop/WindowTop-App#209] [Ideas]
Hello there, how about this then. I noticed that there are some programs (for instance remote desktop software) that minimize the window on the remote desktop when you establish a connection from your end even if the window on the remote desktop is set as always on top window. Would it be useful if we had another button, apart from set top, a never minimize button maybe or incorporate the never minimize in set top button?
I appreciate your time

**Comment by gileli121** [comment]
@Panostis 
I am not sure that I understand but it looks like it should not be solved by WindowTop...
It looks like this should be addressed by the remote desktop app

**Reply by Panostis** [reply]
Yes, you must be right about that! Thanks a lot for pointing out!
